src/Symfony/Bundle/SecurityBundle/Resources/config/security_debug.xml

```diff
@@ -10,5 +10,11 @@
         <service id="debug.security.access.decision_manager" class="Symfony\Component\Security\Core\Authorization\TraceableAccessDecisionManager" decorates="security.access.decision_manager">
             <argument type="service" id="debug.security.access.decision_manager.inner" />
         </service>
+
+        <service id="debug.security.firewall" class="Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener" decorates="security.firewall" public="true">
```

Comment:
done but that means the bundle needs a conflict for the dispatcher as event subscribers must be public before 3.3